runtime/build.gradle.kts

```diff
@@ -2,15 +2,25 @@ import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
 plugins {
     kotlin("multiplatform")
+    id("com.android.library")
     `maven-publish`
 }
 
+repositories {
+    google()
+}
+
 kotlin {
 
-    jvm {
-        withJava()
```

Comment:
I'm not sure why we needed the `withJava()` expression, but this caused a failure after adding the `android` sourceSet, because the Java plugin was being applied.